docs/designers/walkthrough-windows-forms-designer.md

```diff
@@ -269,11 +269,11 @@ The first step is to create the application project. You'll use this project to
 
 1. Create a new **Windows Forms Application** project and name it **DemoCalculatorTest**.
 
-2. In **Solution Explorer**, right-click the **DemoCalculatorTest** project, and then select **Add Reference** to open the **Add Reference** dialog box.
+2. In **Solution Explorer**, select and hold (or right click) the **DemoCalculatorTest** project, and then select **Add Reference** to open the **Add Reference** dialog box.
```

Comment:
The guidance provided here (https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui) indicates that "right-click" needs a hyphen. Please resubmit the PR with that correction, if you would? Thanks.